src/_guides/language/extension-methods.md

```diff
@@ -212,14 +213,30 @@ extension NumberParsing on String {
 ```
 <div class="prettify-filename">lib/string_apis.dart</div>
 
-To create a local extension that's visible only in
-the library where it's declared,
-either omit the extension name or give it a name
-that starts with an underscore (`_`).
-
-The members of the extension can be methods, getters, setters, operators.
+The members of an extension can be methods, getters, setters, or operators.
 Extensions can also have static fields and static helper methods.
 
+### Unnamed extensions
+
+When declaring an extension, you can omit the name.
+Unnamed extensions are visible only
+in the library where they're declared.
+Since they don't have a name,
+there is no way to explicitly apply them
+to resolve [API conflicts](#api-conflicts).
```

Comment:
**cleanup:** This still needs active voice to clean it up.
```suggestion
When declaring an extension, you can omit the name.
An unnamed extension can be used only in the library
where it's declared.
Since they don't have a name,
they cannot be explicitly applied
to resolve [API conflicts](#api-conflicts).
```

**note:** If you say _visible_, you need to say visible to _whom_.

Reply:
I'm rather hesitant to lose the word visible here, "visible inside" or "visible in" a library, package, module, etc. is a common construct in programming languages and their docs, including ours.

Reply:
@parlough : What does "visible" mean? Visible to whom? What does it mean to not be visible? That's what I'm trying to understand.

Reply:
@atsansone By "visible in a library", I mean visible to other code (at least) within the same library, visible to the analyzer and compiler(s) when determining what is accessible in the library, in some-cases visible to reflective accessors of the library through `dart:mirrors`.

If it's visible only to that library (private), it won't show up in the `dart doc` output for that library, other libraries can't access the extension, the analyzer won't suggest it when coding in other libraries.

If it's not private, that means other libraries can import it making its members will be visible and accessible, `dart doc` can include the member in its rendered output, etc.

Essentially, I do think "can be used only in the library where it's declared" covers the primary use case of a developer coding, but it's also about whether its discoverable, included in various outputs, etc.